src/lib/logger.test.ts

```
@@ -1,19 +1,67 @@
-  logger.info('This is an info message - should appear in both dev and production');
-  logger.warn('This is a warning message - should appear in both dev and production');
-  logger.error('This is an error message - should appear in both dev and production');
-  
+
+  mockLogger.debug('This is a debug message - should only appear in development');
+  mockLogger.info('This is an info message - should appear in both dev and production');
+  mockLogger.warn('This is a warning message - should appear in both dev and production');
+  mockLogger.error('This is an error message - should appear in both dev and production');
+
   console.log('Logger test complete. Check logs/ directory for output files.');
 }
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)